src/main/resources/data/events.json

```diff
@@ -55,7 +97,21 @@
       },
       {
         "key": "armor",
-        "summary": "Armor"
+        "summary": "Armor",
+        "values": [
+        	{
+        		"value": 0,
+        		"description": "Energy Shield"
+        	},
+        	{
+        		"value": 1,
+        		"description": "Graphene Armor"
+        	},
+        	{
+        		"value": 2,
+        		"description": "Titanium Armor"
```

Comment:
On second thought, maybe better names for these would be "Purple Armor", "Green Armor" and "Red Armor". Otherwise identifying the right button in the client interface might get quite tricky for players.